include/language.php

```diff
@@ -656,7 +656,7 @@ public function get_locale( $filter = 'raw' ) {
 	 *
 	 * @phpstan-return LanguageData
 	 */
-	public function get_object_vars( $context = 'display' ) {
+	public function to_array( $context = 'display' ) {
```

Comment:
We'll need to do the change in [PLL Pro](https://github.com/polylang/polylang-pro/blob/0bfb34007694cbb989484698bed4e2fe4aa152fb/modules/rest/rest-api.php#L221) too.

Reply:
Yup, I wanted to be sure to merge this change before doing a PR.